PaTTAN Room Displays/PaTTAN Room Displays/App.xaml.cs

```diff
@@ -1,11 +1,28 @@
-﻿namespace PaTTAN_Room_Displays;
+﻿using System.Threading.Tasks;
+
+namespace PaTTAN_Room_Displays;
 
 public partial class App : Application
 {
+	RoomDataService roomDataService = new RoomDataService();
+	public static List<Meeting> meetingList = new List<Meeting>();
 	public App()
 	{
 		InitializeComponent();
+		getData();
 
-		MainPage = new MainPage();
+		if(meetingList.Count > 0)
```

Comment:
There is some whitespace inconsistency (tabs vs. spaces, I assume) throughout this class

Reply:
yes, the code formatter for VS 2022 is broken.  
https://developercommunity.visualstudio.com/t/formatting-doesnt-work-in-vs-2022-preview/1517712

Reply:
I haven't found a good xaml parser in VS Code, but I did find a nice C# parser.  I tried using a XML parser but that was a mistake.   With any luck the maui stuff will get out of preview soon.